aine/parts/footer.html

```diff
@@ -0,0 +1,9 @@
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"80px","bottom":"30px"}}}} -->
+<div class="wp-block-group alignfull" style="padding-top:80px;padding-bottom:30px"><!-- wp:spacer {"height":"80px"} -->
+<div style="height:80px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"0.8rem","letterSpacing":"-0.2px"}}} -->
+<p class="has-text-align-center" style="font-size:0.8rem;letter-spacing:-0.2px">Designed with <a rel="nofollow" href="https://wordpress.org">WordPress</a></p>
```

Comment:
We should allow this string to be translatable, and this file should probably use the included footer pattern.